tic-tac-toe/Referee.py

```diff
@@ -0,0 +1,50 @@
+import re
+
+import MagicNumbers
+import MagicStrings
+
+
+class Referee:
+    def __init__(self):
+        pass
+
+    def check_validity_and_make_move(self, game_board, input_string, move_char):
+        input_string = input_string.strip()
+        is_string_valid = self.check_input(input_string)
+        if not is_string_valid:
+            self.referee_says(MagicStrings.WRONG_INPUT_FORMAT)
+            return False
+        is_valid_move = game_board.check_validity_and_make_move(input_string[0], input_string[2], move_char)
```

Comment:
probably better to convert input_string to actual indexes here and then pass on because UI and business logic will be more decoupled.